### PR TITLE
docs(aio): Add resources contributor info

### PR DESCRIPTION
**Pull request by StephenFluin**
## PR Checklist
Does please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

```
[X] Documentation content changes

```

### CODE REVIEW

**Review by juleskremer — APPROVED** [review]

File: aio/content/marketing/resources-contributing.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Our Resources List
+We maintain a small list of some of the top Angular resources from across the community, stored in `resources.json`. This list is not intended to be comprehensive, but to act as a starting to connect Angular developers to the rest of the community.
+
+# How do I get listed?
+While we can't accept all contributions, qualifying contributions can be submitted via a PR adding yourself to the `resources.json` file. All contributions should be in the appropriate section and must meet at the following criteria:
+
+1. Your contribution must be valid, and contain a link to a page talking specifically about using Angular.
+1. Your contribution should have a clear and concise title and description
```
Comment (gkalpak) [inline aio/content/marketing/resources-contributing.md]: Missing `.` (here and below) :grin:

File: aio/content/marketing/resources-contributing.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1. Your resource should follow our brand guidelines (see our [https://angular.io/presskit](Presskit))
```
Comment (gkalpak) [inline aio/content/marketing/resources-contributing.md]: Didn't you mean `[Presskit](https://angular.io/presskit)`?

File: aio/content/marketing/resources-contributing.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Our Resources List
+We maintain a small list of some of the top Angular resources from across the community, stored in `resources.json`. This list is not intended to be comprehensive, but to act as a starting to connect Angular developers to the rest of the community.
+
+# How do I get listed?
```
Comment (gkalpak) [inline aio/content/marketing/resources-contributing.md]: Douments are not allowed to have multiple `<h1>` elements (anf that's why the `aio` job fails on Travis.

**Review by gkalpak — CHANGES_REQUESTED** [review]
Shouldn't a corresponding entry be added to `navigation.json`? Else this will be hidden in the site, but not easily accessible. (Or is this intended atm?)

File: aio/content/marketing/resources-contributing.md (hunk 3 above)
Comment (StephenFluin) [inline aio/content/marketing/resources-contributing.md]: Correcting to a single H1

File: aio/content/marketing/resources-contributing.md (hunk 1 above)
Comment (StephenFluin) [inline aio/content/marketing/resources-contributing.md]: Removing `.`

File: aio/content/marketing/resources-contributing.md (hunk 2 above)
Comment (StephenFluin) [inline aio/content/marketing/resources-contributing.md]: I did!

**Comment by StephenFluin** [comment]
Not linking to this page from navigation is intentional. The hope is that it is only seen by those looking in the source code and trying to modify `resources.json`.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
> Not linking to this page from navigation is intentional. The hope is that it is only seen by those looking in the source code and trying to modify resources.json.

Just beware that this _will_ show up in search results too :grin:
https://pr17851-69ec333.ngbuilds.io/?search=resources

**Comment by petebacondarwin** [comment]
Cherry picked into 4.2.x as c460757d2ec2bae9276b2fd1f2b5422804b15f53

**Pull request merged**